test/indexSpec.js

```diff
@@ -471,12 +471,137 @@ describe('PluginBells', function () {
       it('returns the plugin\'s prefix', function * () {
         yield assertResolve(this.plugin.getPrefix(), 'example.red.')
       })
+
+      it('fails without any prefix', function (done) {
+        const plugin = new PluginBells({
+          // no prefix
+          account: 'http://red.example/accounts/mike',
+          password: 'mike'
+        })
+        plugin.getPrefix().catch((err) => {
+          assert.equal(err.message, 'Prefix has not been set')
+          done()
+        })
+      })
+
+      it('cannot connect without any prefix', function (done) {
+        const plugin = new PluginBells({
+          account: 'http://blue.example/accounts/mike',
+          password: 'mike',
+          debugReplyNotifications: true,
+          debugAutofund: {
+            connector: 'http://mark.example',
+            admin: {username: 'adminuser', password: 'adminpass'}
+          }
+        })
+
+        const infoRedLedger = cloneDeep(require('./data/infoRedLedger.json'))
+        const nockInfo = nock('http://blue.example')
+          .get('/')
+          .reply(200, infoRedLedger)
+
+        const nockAccount = nock('http://blue.example')
+          .get('/accounts/mike')
+          .reply(200, {
+            ledger: 'http://blue.example',
+            name: 'mike'
+          })
+
+        plugin.connect().catch((err) => {
+          assert.equal(err.message, 'Unable to set prefix from ledger or from local config')
+          nockInfo.done()
+          nockAccount.done()
+          done()
```

Comment:
For the future you may want to set this up as 
```js
fn.catch((err) => {
  assert.equal(err.message, 'blah')
}).catch(done)
```
because the way it's set up now if one of those assertions throws you'll see the test failing because of a timeout, because done will never be called. It's more helpful if it prints the actual error